wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -17,8 +17,41 @@ You'll also display the balance of an ERC-20 token and perform a token transfer.
 
 ## 1. Set up the project
```

Comment:
If we are to repeat the same steps from the Connect to Starknet guide here (which I'm fine with), can we use the same / similar wording for consistency?